apache2/sites-enabled/25-rt.ocf.berkeley.edu.conf

```diff
@@ -30,7 +30,7 @@
   </Location>
 
   <Location "/REST/1.0">
-    Require ip 127.0.0.1 ::1 169.229.226.35 2607:f140:8801::1:35 169.229.226.36 2607:f140:8801::1:36 169.229.226.7 2607:f140:8801::1:7 169.229.226.10 2607:f140:8801::1:10 169.229.226.12 2607:f140:8801::1:12 169.229.226.14 2607:f140:8801::1:14 169.229.226.16 2607:f140:8801::1:16
+    Require ip 127.0.0.1 ::1 172.17.0.0/16 169.229.226.35 2607:f140:8801::1:35 169.229.226.36 2607:f140:8801::1:36 169.229.226.7 2607:f140:8801::1:7 169.229.226.10 2607:f140:8801::1:10 169.229.226.12 2607:f140:8801::1:12 169.229.226.14 2607:f140:8801::1:14 169.229.226.16 2607:f140:8801::1:16
```

Comment:
what is this 172.17 range?

Reply:
When the healthcheck tries to contact the endpoint using the `$HOST0:$PORT` address that Marathon gives it, it uses a `172.17.x.x` IP as the source address.

Do you think it's better to not include this and make the endpoint address `localhost:80` instead?